### ISSUE TITLE
Bulk entry for Labels and annotations does not work

### ISSUE THREAD

**Issue by sowmyav27** [rancher/dashboard#1066]
**On master-head - commit id: `f08bf21f9`**

- Create a Config map
- Enter this in labels/annotations's key field
```
key1=value1
key2=value2
key3=value3
```
- Bulk entry does not work as said
<img width="888" alt="Screen Shot 2020-08-20 at 4 23 26 PM" src="https://user-images.githubusercontent.com/26032343/90835485-960b0b00-e301-11ea-81c3-6df9df0b4772.png">

**Comment by mantis-toboggan-md** [comment]
This appears to be working now, after this commit: https://github.com/rancher/dashboard/commit/726f680c1f98ca3bf7c397a6e48487f99461e00d

**Comment by sadiapoddar** [comment]
verified the fix on 2.5-head, 3d7ffc99
Steps

1. Created a Config map
2. Enter this in labels/annotations's key field

```
key1=value1
key2=value2
key3=value3

```

<img width="2097" alt="Screen Shot 2020-12-11 at 5 16 05 PM" src="https://user-images.githubusercontent.com/72409237/101968663-1d43fd80-3bd5-11eb-9e00-6b49378dfae1.png">

Successfully created the config map, closing the bug the fix is working.
